### PR TITLE
Do not set last retired fed p2sh script in new commitFederation impl

### PR DESCRIPTION
**Pull request by julia-zack**
Last-retired-federation script shouldn't be set in new commitFederation implementation, since we will set the to-be-active-federation after its validation.

### CODE REVIEW

**Review by jeremy-then — APPROVED** [review]
LGTM.

**Review by apancorb — APPROVED** [review]
LGTM

**Review by nathanieliov — APPROVED** [review]

File: rskj-core/src/main/java/co/rsk/peg/federation/FederationSupportImpl.java [hunk 1]
```diff
@@ -733,12 +734,11 @@ private FederationChangeResponseCode commitPendingFederation(PendingFederation c
         // set proposed federation
         Federation proposedFederation = buildFederationFromPendingFederation(currentPendingFederation);
         provider.setProposedFederation(proposedFederation);
+        setNewActiveFederationCreationBlockHeight();
```
Comment (marcos-iov) [inline rskj-core/src/main/java/co/rsk/peg/federation/FederationSupportImpl.java:737]: Should this be done at this point? Shouldn't we set this value after the proposed federation is validated? If the validation fails we'd end up with an invalid value here, right?

Comment (julia-zack) [reply rskj-core/src/main/java/co/rsk/peg/federation/FederationSupportImpl.java:737]: we discussed this when developing phase1, the idea was that the activation period remain the same. But yes it makes sense to do it after the validation.

Comment (julia-zack) [reply rskj-core/src/main/java/co/rsk/peg/federation/FederationSupportImpl.java]: updated

**Review by marcos-iov — APPROVED** [review]

**Pull request merged**